joomla/joomla-cms
# Remove console.log from tinymce.min.js

Pull Request for Issue #14937

### Summary of Changes

Remove the console log from `tinymce.min.js`


### Testing Instructions

- Edit content in the backend
- Ensure nothing is dumped to the browser console
